local/network.go

```diff
@@ -576,11 +594,12 @@ func (ln *localNetwork) RemoveNode(nodeName string) error {
 	return ln.removeNode(nodeName)
 }
 
-// Assumes [net.lock] is held
+// Assumes [ln.lock] is held
 func (ln *localNetwork) removeNode(nodeName string) error {
-	if ln.isStopped() {
+	if ln.stopped {
```

Comment:
this can be avoided in the check isStopped() is moved to RemoveNode. Same for all other public functions.

Reply:
moved